packages/contracts/src/utils/bytes32.ts

```diff
@@ -107,20 +114,13 @@ export function subaccountFromHex(subaccount: string): Subaccount {
   return subaccountFromBytes32(getBytes(subaccount));
 }
 
-export function strToBytes(input: string, bytesLen: number): Bytes {
```

Comment:
Decided to combine this into `subaccountNameToBytes12` as we have no other usage of it. We can then use `subaccountNameToBytes12` in `subaccountToBytes32` so that we can share the custom `isHexString` logic